stats.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict';
 
+var async = require('async');
```

Comment:
if you care about speed and CPU load, you might get some benefit by using https://github.com/mcollina/steed.